iree/compiler/Dialect/Flow/Transforms/test/matmul_to_mmt4d.mlir

```diff
@@ -70,3 +70,57 @@ func @check_mmt4d_with_init_tensor_and_fill(%arg0: tensor<24x8xf32>, %arg1: tens
 // CHECK-SAME:   tensor<8x32xf32> into tensor<4x2x8x4xf32>
 //      CHECK: %[[DST_INIT:.+]] = linalg.init_tensor [3, 8, 8, 4] : tensor<3x8x8x4xf32>
 //      CHECK: [[DST:.+]] linalg.fill(%[[ZERO:.+]], %[[DST_INIT]])
+
+// -----
+func @check_mmt4d_i8_static_pad(%arg0: tensor<3x5xi8>, %arg1: tensor<5x2xi8>, %arg2: tensor<3x2xi32>) -> tensor<3x2xi32> {
+    %0 = linalg.matmul ins(%arg0, %arg1 : tensor<3x5xi8>, tensor<5x2xi8>) outs(%arg2 : tensor<3x2xi32>) -> tensor<3x2xi32>
+    return %0 : tensor<3x2xi32>
+}
+//      CHECK: @check_mmt4d_i8_static_pad(%[[LHS:.+]]: tensor<3x5xi8>, %[[RHS:.+]]: tensor<5x2xi8>, %[[ACC:.+]]: tensor<3x2xi32>)
```

Comment:
Recommend using `CHECK-LABEL` for function names and then a `CHECK-SAME` for args. `CHECK-LABEL` clears the defined variables because otherwise you'll have the ones from the previous test case!

```suggestion
//      CHECK-LABEL: @check_mmt4d_i8_static_pad(
//      CHECK-SAME:     %[[LHS:.+]]: tensor<3x5xi8>, %[[RHS:.+]]: tensor<5x2xi8>, %[[ACC:.+]]: tensor<3x2xi32>)
```